release-notes/kvm/v11.3.0.md

```diff
@@ -0,0 +1,19 @@
+## :zap: Giant Swarm Release 11.3.0 for KVM :zap:
+
+This release includes Kubernetes v1.16.9 as well as reliability and user experience improvements.
+
+In addition, this release replaces CoreOS with Flatcar Container Linux.
+CoreOS has gone [end-of-life](https://coreos.com/os/eol/) and is being rapidly phased out.
+Flatcar is a compatible fork of CoreOS which receives ongoing support.
+To continue receiving security updates and to minimize the effort needed to migrate in the future, we recommend upgrading to this release.
+
+Below, you can find more details on components that were changed with this release.
+
+### Kubernetes [v1.16.9](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v1168)
+- Updated from v1.16.3.
+
+### kvm-operator [v3.11.0](https://github.com/giantswarm/kvm-operator/releases/tag/v3.11.0)
+- Use Flatcar linux instead of CoreOS.
+- Streamlined image templating for core components for quicker and easier releases in the future.
+- Support setting OIDC username and groups prefix.
+- Enabled per-cluster configuration of kube-proxy's `conntrackMaxPerCore` parameter.
```

Comment:
Should we have a changelog for Flatcar? The Brazil timezone?

Reply:
Good point - I would add the flatcar changelog from `9.0.1`:
https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v9.0.1.md#flatcar-linux-234531

No brazil timezone I would say :D

Reply:
Makes sense. From a customer perspective I might be interested in knowing the new kernel version (4.19.68->4.19.107) and OpenSSL, but the rest isn't very relevant.